exonum/src/api/node/public/system.rs

```diff
@@ -109,11 +120,9 @@ impl SystemApi {
         let out_conn = self.shared_api_state.outgoing_connections().len();
 
         if in_conn == 0 && out_conn == 0 {
-            ConnectivityStatus::NotConnected
+            ConnectivityStatus::not_connected()
         } else {
-            ConnectivityStatus::Connected(PeersAmount {
-                amount: in_conn + out_conn,
-            })
+            ConnectivityStatus::connected(in_conn + out_conn)
```

Comment:
Don't know what are these connections, but does the sum of incoming and outgoing connections correspond to the _number of peers_ or of connections? Can't we have both an ingoing and outcoming connection to the same node?

Reply:
@dmitry-timofeev No, we can't.

Reply:
I'd add a comment in this case, because it is far from immediately apparent.